src/main/java/com/carhouse/controller/LoginController.java

```diff
@@ -0,0 +1,38 @@
+package com.carhouse.controller;
+
+import com.carhouse.model.User;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+/**
+ * Login controller.
+ */
+@Controller
+public class LoginController {
```

Comment:
Do you have the requirement to create login page?

Reply:
I had a time so I decided to make pages what I will need in a feature.